ios/Lift/LiftServer/ExerciseModel+Marshalling.swift

```diff
@@ -31,13 +31,19 @@ extension Exercise.SessionProps {
 extension Exercise.Exercise {
     
     static func unmarshal(json: JSON) -> Exercise.Exercise {
-        return Exercise.Exercise(name: json["name"].stringValue, intensity: json["intensity"].double)
+        return Exercise.Exercise(name: json["name"].stringValue, intensity: json["intensity"].double, metric: Exercise.Metric(value: json["metric"]["value"].doubleValue, metricUnit: json["metric"]["metricUnit"].stringValue))
```

Comment:
It'd be better if you added ``unmarshal(json: JSON) -> Exercise.MetricUnit`` to its own extension and use it here.